routers/web/repo/wiki.go

```diff
@@ -93,17 +93,32 @@ func findEntryForFile(commit *git.Commit, target string) (*git.TreeEntry, error)
 }
 
 func findWikiRepoCommit(ctx *context.Context) (*git.Repository, *git.Commit, error) {
-	wikiRepo, err := gitrepo.OpenWikiRepository(ctx, ctx.Repo.Repository)
-	if err != nil {
-		ctx.ServerError("OpenRepository", err)
-		return nil, nil, err
+	wikiGitRepo, errCommit := gitrepo.OpenWikiRepository(ctx, ctx.Repo.Repository)
+	if errCommit != nil {
+		ctx.ServerError("OpenRepository", errCommit)
+		return nil, nil, errCommit
```

Comment:
Wrong error name for that method?

Reply:
Not that wrong. It is also for "getting the commit". 

The error handling is not ideal here, but I don't want to make it unnecessary complex, so just user different error names here.

Will rename it to something else.